lib/lib.go

```diff
@@ -46,8 +46,8 @@ var (
 	log = golog.Logger("lib")
 )
 
-// VersionNumber is the current version qri
-const VersionNumber = "0.9.2-dev"
+// VersionNumber is the current qri version
+const VersionNumber = "0.9.1"
```

Comment:
Okay, I was confused by this at first by this change, decrementing the version number. Turns out this commit a60187f8d4a0f6af3ae13605fa85fb218b2463e5 changed 0.9.1-alpha into 0.9.2-dev. Presumably we were using 0.9.1-alpha because we were intending on cutting 0.9.1 a few weeks ago, but that never happened, then this commit mistakingly skipped it. So we're doing this release as 0.9.1.